client/fileupload.go

```diff
@@ -100,9 +113,24 @@ func (fu *FileUpload) HandleOperation(operation string, payload []byte) *ErrorRe
 
 // OnTick triggers periodic file uploads. Invoked from the periodic executor in AutoUploadable
 func (fu *FileUpload) OnTick() {
-	err := fu.DoTrigger(fu.uploadable.nextgUID(), nil)
+	err := fu.DoTrigger(fu.uploadable.nextUID(), nil)
 
 	if err != nil {
 		logger.Errorf("error on periodic trigger: %v", err)
 	}
 }
+
+func (fu *FileUpload) isGlobUploadPermitted(glob string) (bool, error) {
```

Comment:
What is the reason to return bool and error? It seems that just a single error should be enough.

Reply:
filepath.Match(fu.filesGlob, glob) can return an error, I suppose this is the reason